icaruscode/TPC/ICARUSWireCell/detsimmodules_wirecell_ICARUS.fcl

```diff
@@ -85,4 +85,8 @@ icarus_simwire_wirecell_fitSR.wcls_main.structs.gain0: 11.9918701 # mV/fC
 icarus_simwire_wirecell_fitSR.wcls_main.structs.gain1: 12.6181926 # mV/fC
 icarus_simwire_wirecell_fitSR.wcls_main.structs.gain2: 13.0261362 # mV/fC
 icarus_simwire_wirecell_fitSR.wcls_main.structs.shaping1: 1.45 # us
+
+icarus_simwire_wirecell_filtersed: @local::icarus_simwire_wirecell
+icarus_simwire_wirecell_filtersed.wcls_main.configs: ["pgrapher/experiment/icarus/wcls-multitpc-sim-drift-simchannel-refactored-filtersed.jsonnet"] 
+
```

Comment:
I think we either want this to be a new fcl `detsimmodules_wirecell_filtered_ICARUS.fcl` or the `detsim_2d_filter.fcl` should happen in the base fcl

Reply:
Provided that I understood the logic correctly, I also think it's better to have the fhicl with icarus settings for wire cell separated from that containing the filtering otherwise people might get confused and use the wrong one. 
My proposal would be to have 
- detsimmodules_wirecell_ICARUS.fcl
- detsimmodules_wirecell_filtered_ICARUS.fcl (that recalls the former and adds what's currently in lines 89-90)

Reply:
I think the best thing here is to include this change as part of the`detsim_2d_filter.fcl` that way I just change the one wirecell piece there and we don't need this. That should work fine for the variation based approach. 

Essentially add:`icarus_simwire_wirecell.wcls_main.configs: ["pgrapher/experiment/icarus/wcls-multitpc-sim-drift-simchannel-refactored-filtersed.jsonnet"]` to `detsim_2d_filter.fcl` instead of here

Reply:
This solution works. thanks